macOS/DuckDuckGo/DataImport/ChromiumPreferences.swift

```diff
@@ -77,4 +94,23 @@ struct ChromiumPreferences: Decodable {
         profile.createdByVersion ?? extensions?.lastChromeVersion ?? extensions?.lastOperaVersion
     }
 
+    var isNewTabShortcutsEnabled: Bool {
+        ntp?.shortcustVisible ?? false
+    }
+
+    /// Returns `true` if the New Tab Page is using auto-generated shortcuts.
+    /// Returns `false` if the New Tab Page is using custom shortcuts.
+    var isUsingAutoGeneratedShortcuts: Bool {
```

Comment:
not a big deal, but if it requires a comment to describe what is the boolean variable about, maybe consider introducing a enum in place of it:
```
enum NewTabPageShortcutsStyle {
  case autoGenerated, custom
}
var newTabPageShortcutsStyle: NewTabPageShortcutsStyle {
  (ntp?.useMostVisitedTiles ?? false) ? .autoGenerated : .custom
}
```